.github/workflows/run-examples.yaml

```diff
@@ -0,0 +1,71 @@
+name: Run Examples
+on:
+  workflow_call:
+    inputs:
+      sdk_version:
+        description: 'SDK version to run examples for'
+        required: false
+        type: string
+        default: 'LATEST'
+      jdk:
+        description: 'JDK version to use'
+        required: true
+        type: string
+        default: '17'
+      sdk_generation_workflow_run_id:
+        description: 'ID of the SDK generation workflow run'
+        required: false
+        type: string
+        default: ''
+    secrets:
+      KEY:
+        description: 'API Key'
+        required: true
+      SECRET:
+        description: 'API Key'
+        required: true
+
+jobs:
+  run-examples:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        if: ${{ inputs.sdk_generation_workflow_run_id != '' }}
+        with:
+          repository: 'ExpediaGroup/rapid-java-sdk'
+      - uses: dawidd6/action-download-artifact@v6
+        if: ${{ inputs.sdk_generation_workflow_run_id != '' }}
+        with:
+          run_id: ${{ inputs.sdk_generation_workflow_run_id }}
+          name: jar
+          path: examples/jar
+      - uses: dawidd6/action-download-artifact@v6
+        if: ${{ inputs.sdk_generation_workflow_run_id != '' }}
+        with:
+          run_id: ${{ inputs.sdk_generation_workflow_run_id }}
+          name: sdk
+          path: examples/sdk
+      - name: Set up JDK
+        uses: actions/setup-java@v4
+        with:
+          java-version: ${{ inputs.jdk }}
+          distribution: 'corretto'
+      - name: Extract POM from JAR
```

Comment:
Many of these steps are done because of the temp situation that the POM file in the main repo still references a parent POM file and you cannot point to it here, as we previously discussed, (right?)

Can you please create a ticket to replace this logic once we completely retire the parent POM?

Reply:
To be discussed offline.